packages/dd-trace/src/config.js

```diff
@@ -275,6 +275,11 @@ ken|consumer_?(?:id|key|secret)|sign(?:ed|ature)?|auth(?:entication|orization)?)
     )
 
     const remoteConfigOptions = options.remoteConfig || {}
+    const DD_REMOTE_CONFIGURATION_ENABLED = coalesce(
+      process.env.DD_REMOTE_CONFIGURATION_ENABLED,
+      !process.env.AWS_LAMBDA_FUNCTION_NAME,
```

Comment:
I think we should use the variable `inAWSLambda` that's defined a bit below

Reply:
updated accordingly. Thank you.